cns/service/main.go

```diff
@@ -1182,34 +1184,41 @@ func InitializeCRDState(ctx context.Context, httpRestService cns.HTTPService, cn
 	}
 	logger.Printf("reconciled initial CNS state after %d attempts", attempt)
 
-	// the nodeScopedCache sets Selector options on the Manager cache which are used
+	scheme := kuberuntime.NewScheme()
+	if err := corev1.AddToScheme(scheme); err != nil { //nolint:govet // intentional shadow
```

Comment:
what is shadowing this?

Reply:
it shadows an earlier declaration of `err` (and did before, also, but the linter is blocking it now). not a problem because this declaration is scoped to this block.